libraries/testkit/cluster.py

```diff
@@ -188,8 +201,27 @@ def reset(self, sg_config_path):
             else:
                 num_replicas = get_sg_replicas(self._cluster_config)
                 playbook_vars["num_index_replicas"] = '"num_index_replicas": {},'.format(num_replicas)
+
+            if is_x509_auth(self._cluster_config):
+                playbook_vars[
+                    "certpath"] = '"certpath": "/home/sync_gateway/certs/chain.pem",'
+                playbook_vars[
+                    "keypath"] = '"keypath": "/home/sync_gateway/certs/pkey.key",'
+                playbook_vars[
+                    "cacertpath"] = '"cacertpath": "/home/sync_gateway/certs/ca.pem",'
+                playbook_vars["server_scheme"] = "couchbases"
+                playbook_vars["server_port"] = ""
```

Comment:
doesn't it need 18091 port as that is for https

Reply:
No, x509 certs doesn't need port.